utils/config.ts

```diff
@@ -0,0 +1,66 @@
+import { Category } from './poco-tools';
+import json from '../config/config.json';
+
+const config = json as Config;
+
+function isNativeChain(chain?: ChainConfig) {
+    chain = chain ?? config.chains.default;
+    return chain.asset === 'Native' || process.env.TEST__IS_NATIVE_CHAIN === 'true';
+}
+
+/**
+ * Get the config of the current chain or throw if it is not defined.
+ */
+function getChainConfig(chainId: bigint): ChainConfig {
+    const chainConfig = config.chains[chainId.toString()];
+    if (!chainConfig) {
+        throw new Error(`Chain config undefined for chain ${chainId}`);
+    }
+    return chainConfig;
+}
+
+function getChainConfigOrDefault(chainId: bigint): ChainConfig {
+    return config.chains[chainId.toString()] ?? config.chains.default;
+}
+
+type Config = {
+    categories: Category[];
+    registriesBaseUri: {
+        app: string;
+        dataset: string;
+        workerpool: string;
+    };
+    chains: {
+        [key: string]: ChainConfig;
+    };
+};
+
+type ChainConfig = {
+    _comment: string;
+    asset: string;
+    token?: string | null;
+    uniswap?: boolean;
+    etoken?: string;
```

Comment:
Now-or-future note, I think we can remove `etoken`

Reply:
I agree we need to remove it but it's better to do in the PR were we merge/clean all 3 json config files.